inc/request.h

```diff
@@ -102,6 +102,9 @@ typedef struct RequestParams
 
     // Data passed to the callbacks
     void *callbackData;
+
+    // Request timeout. 0 if not set
```

Comment:
I think this "0 if not set" comment is inaccurate and confusing?  Or maybe I just don't understand?  What about "If 0, no timeout will be enforced"?